Markdown

amanex007 opened the pull request:
Git offers a better readme file in .md extension that is markdown extention

amanex007 committed:
Markdown

Git offers a better readme file in .md extension that is markdown extention

amanex007 commented:
I have been a part of Google code in this year and learnt some things, i would love to contribute to this project.

pydeveloperashish commented:
> Git offers a better readme file in .md extension that is markdown extention

Added

pydeveloperashish commented:
> I have been a part of Google code in this year and learnt some things, i would love to contribute to this project.

Sure

amanex007 committed:
Create a ReadME.md With the instructions

I believe this will be a better instruction to follow, since normally readme contain the how to do installation part of the project.

amanex007 committed:
Merge branch 'master' into patch-2
